SDKGenerator.njsproj

```diff
@@ -93,7 +93,7 @@
   <PropertyGroup Condition=" '$(Configuration)' == 'PlayFabGameServer' ">
     <DebugSymbols>true</DebugSymbols>
     <OutputPath>bin\</OutputPath>
-    <ScriptArguments>csharp-unity-gameserver=..\sdks\PlayFabGameServer -apiSpecPath -buildIdentifier UnitySDK_manual</ScriptArguments>
+    <ScriptArguments>csharp-unity-gameserver=..\sdks\PlayFabGameServer -apiSpecPath -buildIdentifier UnitySDK_manual -verticalName testVerticalName</ScriptArguments>
```

Comment:
>-verticalName testVerticalName [](start = 117, length = 30)

Do you need to keep this here by default?

Reply:
DOH, nope.
Thanks!